### DISCUSSION TITLE
Implement Signup on elk

### DISCUSSION THREAD

**Discussion by visv-in** [elk-zone/elk#2512] [General]
Hi, I would like to know if there are any legal restrictions to implement mastodon signup on elk.zone UI. I also want to know the possibility of  signup being implemented in elk (In this repository).

**Comment by patak-cat** [comment]
We can't do it for elk.zone as it is a generic client for any instance (by default we show the posts from https://m.webtoo.ls if you aren't signed in with an user, but that is it).
You could check what Mozilla Social did to implement sign up for https://mozilla.social in their fork https://github.com/MozillaSocial/elk

**Reply by Shinigami92** [reply]
We could do it, if we would provide a select field / autocomplete field to select a "provider" / instance.
But I'm not sure if it would be simpler to implement just a link to https://joinmastodon.org
